StudentManagement/StudentManagement.Console/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using StudentManagement.Console;
+
+List<Student> students = new List<Student>();
+students.Add(new Student());
+Console.WriteLine("Enter student name");
+students[0].Name = Console.ReadLine();
+Console.WriteLine("Enter student subname");
+students[0].Surname = Console.ReadLine();
+Console.WriteLine("What the student is studying");
+students[0].Occupation = Console.ReadLine();
+Console.WriteLine("Enter student id"); students[0].Id = int.Parse(Console.ReadLine());
```

Comment:
1. Add new code to new line. 
2. Id should be autogenerated!

Reply:
1.  I removed this code, its no longer necessary to have it
2. ID is autogenerated now